webring/README.md

```diff
@@ -0,0 +1,116 @@
+# Webring Plugin
+
+This plugin retrieves the latest posts from a list of web feeds and makes
+them available in templates, effectively creating a [partial webring]. Posts
+are sorted from newer to older.
+
+It is inspired in [openring](https://git.sr.ht/~sircmpwn/openring), a tool for
+generating an html file to include in your [SSG] from a template and a list of
```

Comment:
html → HTML. Also, to be a link, the `[SSG]` needs to be changed to `[SSG][]` (same issue as above).

Reply:
Fixed html lowercase. The link is also _reference link_.